core/pom.xml

```diff
@@ -35,9 +35,9 @@
   <url>http://spark.apache.org/</url>
   <dependencies>
     <dependency>
-      <groupId>com.twosigma</groupId>
+      <groupId>twosigma</groupId>
       <artifactId>cook-jobclient</artifactId>
-      <version>0.1.0</version>
+      <version>0.1.2-SNAPSHOT</version>
```

Comment:
Can we have cook tag an explict release? I'm pretty worried by putting SNAPSHOT versions into the POM.